doc/users/next_whats_new/2017-12-27_legend_title_size_rc.rst

```diff
@@ -0,0 +1,6 @@
+Legend Title Size rc parameter
+---------------------
+
+A new rc parameter has been added as an option in your matplotlibrc allowing you to control the font size of the legend title.
```

Comment:
Looks like this is longer than 79 characters, so is failing the PEP8 tests - please could you make it <= 79 characters on each line?

Reply:
We should not be pep8 checking the rst files....

If we are that seems like a bug in the tests.

Reply:
I think @dstansby misread something; pep8 is failing in `test_legend.py`:
```
/home/travis/build/matplotlib/matplotlib/lib/matplotlib/tests/test_legend.py:193:1: E302 expected 2 blank lines, found 1
```